mkt/settings.py

```diff
@@ -180,7 +180,6 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'mkt.api.middleware.APIBaseMiddleware',
     'mkt.api.middleware.AuthenticationMiddleware',
-    'commonware.log.ThreadRequestMiddleware',
```

Comment:
And actually, I just realized that's not true and `commonware.log` actually depends on the middleware to work :( I'll revert that part.

Worse, I'll need to expose a fake username property to keep the log stuff working :(